pkg/core/interop_system.go

```diff
@@ -530,6 +531,11 @@ func contractCallExInternal(ic *interop.Context, v *vm.VM, h []byte, method stac
 		v.Jump(v.Context(), md.Offset)
 	}
 
+	md = cs.Manifest.ABI.GetMethod(manifest.MethodInit)
```

Comment:
We also shouldn't allow to call `init` method and other methods which starts with `_` (see https://github.com/neo-project/neo/blob/master/src/neo/SmartContract/ApplicationEngine.Contract.cs#L118). This check should be added at the start of `contractCallExInternal`.

Reply:
Yes, fixed.